From 96448567b03f88d1596f67bba1e1983791936866 Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Mon, 10 Jun 2019 16:54:57 +0200
Subject: [PATCH 4/5] Snap: Make antibanding related changes to APIV2 as well

 * Mimic the behavior for APIV1:
   Antibanding is very important for UX with regard to photographing in
artificial lighting, so don't provide users with an option to turn
antibanding off.

Change-Id: Idf78b15ced985dce58fa25056e8d8730c74047fc
---
 res/values/camera2arrays.xml | 4 +---
 res/values/qcomstrings.xml   | 1 -
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/res/values/camera2arrays.xml b/res/values/camera2arrays.xml
index 484b0b7e3..8121f8c38 100755
--- a/res/values/camera2arrays.xml
+++ b/res/values/camera2arrays.xml
@@ -897,14 +897,12 @@ for time lapse recording -->
         <item>0</item>
         <item>1</item>
         <item>2</item>
-        <item>3</item>
     </string-array>
 
     <string-array name="pref_camera2_anti_banding_level_entries" translatable="true">
-        <item>@string/pref_camera2_anti_banding_level_entry_off</item>
+        <item>@string/pref_camera2_anti_banding_level_entry_auto</item>
         <item>@string/pref_camera2_anti_banding_level_entry_50hz</item>
         <item>@string/pref_camera2_anti_banding_level_entry_60hz</item>
-        <item>@string/pref_camera2_anti_banding_level_entry_auto</item>
     </string-array>
 
     <string-array name="pref_camera2_lensshading_mode_entryvalues" translatable="false">
diff --git a/res/values/qcomstrings.xml b/res/values/qcomstrings.xml
index 65165cfa2..e2df51487 100755
--- a/res/values/qcomstrings.xml
+++ b/res/values/qcomstrings.xml
@@ -1063,7 +1063,6 @@
 
     <string name="pref_camera2_anti_banding_level_title">Antibanding</string>
     <string name="pref_camera2_anti_banding_level_default" translatable="false">0</string>
-    <string name="pref_camera2_anti_banding_level_entry_off">Off</string>
     <string name="pref_camera2_anti_banding_level_entry_50hz">50 Hz (Europe)</string>
     <string name="pref_camera2_anti_banding_level_entry_60hz">60 Hz (USA)</string>
     <string name="pref_camera2_anti_banding_level_entry_auto">Auto</string>
-- 
2.17.1

